JS/Lesson13/sinan/script.js

```diff
@@ -2,143 +2,177 @@
 // Declare a variable y and initialize it to 3.
 // Declare a variable z and initialize it to the sum of x and y.
 // var, const, let
+
 const x = 5;
 const y = 3;
-const z = x + y;
-console.log('Exercise 1:', z);
+const z = x+y;
+console.log('Exercises1 :', z);
 
-console.log('------------------------');
-const name = "onur";
-const age = 32;
-const phrase = "Hello world!";
-const myAge = `${phrase} My name is ${name}. I'm ${age} years old.`;
-const myAge2 = phrase + " My name is " + name + ". I'm " + age + " years old.";
 
-console.log('Exercise 2:',myAge);
-console.log('Exercise 2.2:',myAge2);
+console.log('------------------------');
 
 // ---
 // 2. Declare a variable name and initialize it to your name as a string.
 // Declare a variable age and initialize it to your age as a number.
 // Declare a variable phrase and initialize it to the string "Hello, world!".
 // Add name and age to the sentence above. "Hello, world! My name is (name). I'm (age) years old.".
 
+
+const myName='sinan';
+const myAge= 28;
+const phrase ='Hello, world!';
+const salute = '${phrase} My name is ${myName} Im ${myAge} years old.';
+console.log('Exercises2 :', salute);
+console.log(phrase + ' My name is '+ myName +'. Im ' + myAge + ' years old.');
+
+
 console.log('------------------------');
+
 // ---
 // 3. Declare a variable number1 and initialize it to 10.
 // Declare a variable number2 and initialize it to 3.
 // Declare a variable result and initialize it to the quotient of number1 divided by number2.
 // Declare a variable number3 and initialize it to the difference between number1 and number2.
+
 const number1 = 10;
 const number2 = 3;
-const result = number1 / number2;
+const result2 = number1 / number2;
+const number3  = number1 - number2;
+console.log('Exercises3.1 :', result2);
+console.log('Exercises3.2 :',  number3);
 
-const number3 = number1 - number2; 
-
-console.log('Exercise 3.1:',result);
-console.log('Exercise 3.2:',number3);
 
 console.log('------------------------');
+
 // ---
 // 4. Declare a variable firstName and initialize it to your first name as a string.
 // Declare a variable lastName and initialize it to your last name as a string.
 // Declare a variable fullName and initialize it to the concatenation of firstName and lastName.
 
-const firstName = "Salih";
-const lastName = "Kus";
-const fullName = firstName + " " + lastName;
-console.log('Exercise 4:', 'My Full Name is', fullName);
+const firstName = "sinan";
+const lastName = "dagli";
+const fullName = firstName +  lastName;
```

Comment:
In real life, you probably always want a space between first and last name:
```suggestion
const fullName = firstName + " " + lastName;
```